blog/zh/blog/2025/12/19/apisix-ingress-controller-2-0-release.md

```diff
@@ -0,0 +1,90 @@
+---
+title: "Apache APISIX Ingress Controller 2.0：为现代流量管理而生"
+authors:
+  - name: Yilia Lin
+    title: Technical Writer
+    url: https://github.com/Yilialinn
+    image_url: https://github.com/Yilialinn.png
+keywords:
+  - Ingress Controller
+  - APISIX Ingress Controller
+  - Kubernetes Ingress
+  - Gateway API
+  - Apache APISIX
+description: Apache APISIX Ingress Controller 2.0 正式发布，带来全面的 Gateway API 支持、灵活的多数据面部署模式以及无需 etcd 的轻量级运维，以实现更稳健、可扩展的 Kubernetes 流量管理。
+tags: [Ecosystem]
+image: https://static.apiseven.com/2022/10/19/634f6677742a1.png
+---
+
+> Apache APISIX Ingress Controller 2.0 正式发布，带来全面的 Gateway API 支持、灵活的多数据面部署模式以及无需 etcd 的轻量级运维，以实现更稳健、可扩展的 Kubernetes 流量管理。
+
+<!--truncate-->
+
+基于高性能 API 网关 Apache APISIX 构建的 [APISIX Ingress Controller](https://github.com/apache/apisix-ingress-controller)，历经多个版本的迭代与验证，如今已能从容应对大规模流量管理的挑战。我们很高兴地宣布 [APISIX Ingress Controller 2.0](https://apisix.apache.org/zh/docs/ingress-controller/overview/) 正式发布。此版本围绕**全面的兼容性**、**灵活的架构**与**企业级的稳定性**三大核心进行增强，致力于帮助用户平稳、高效地完成技术栈迁移。
+
+## Apache APISIX Ingress Controller 2.0 核心亮点
+
+### 全面支持 Gateway API
+
+本次发布通过新增对 TCPRoute、UDPRoute、GRPCRoute 和 TLSRoute 的支持，实现了支持 Gateway API 的重要里程碑。这些扩展为从传统 HTTP、TCP/UDP 到现代 gRPC 及 TLS 透传/终止等多种流量类型，提供了原生且感知协议的路由能力。通过这种统一的支持，企业能够在一个一致、面向未来的配置模型中管理复杂的入口需求，简化多协议部署，并平滑迈向完整的 Gateway API 体系。
+
+### 引入 Gateway API 扩展
+
+APISIX Ingress Controller 2.0 引入了一套在 `apisix.apache.org/v1alpha1` 组下的官方 Gateway API 扩展。这些扩展由 Gateway API 社区开发和维护，以安全可靠的方式提供生产级能力，丰富了标准规范。
```

Comment:
This is developed and maintained by the `apisix-ingress-controller `extension, not by the gateway API community.